### DISCUSSION TITLE
Issue task lists aren't formatting correctly

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#4952] [Mobile]
The issue can be seen in [this](https://github.com/hwittenborn/tap/issues/7) issue's task list.

The second, third, fourth, fifth and sixth items should be indented by a single level, but they're not being indented at all. The rest of the indentation is also following suit, with items that are indented by two levels only being indented by one.

**Comment by eliperkins** [comment]
Thanks for this bug report @hwittenborn! 🐛  Linking to that issue made this bug quite easy to reproduce, thanks a ton for that!

I've put together your notes in a bug report for our engineering team. I'll let you know when this bug has been fixed. 🙇
